charts/kubernetes-operations/plugindefinition.yaml

```
@@ -9,15 +9,15 @@ kind: PluginDefinition
 metadata:
   name: kubernetes-operations
 spec:
-  version: 1.1.3
+  version: 1.1.4
   displayName: Kubernetes operations bundle
   description: Operations bundle for Kubernetes
   docMarkDownUrl: https://raw.githubusercontent.com/cloudoperators/kubernetes-operations/main/README.md
   icon: https://raw.githubusercontent.com/cloudoperators/kubernetes-operations/main/charts/kubernetes-operations/kubernetes-logo.png
   helmChart:
     name: kubernetes-operations
     repository: oci://ghcr.io/cloudoperators/kubernetes-operations/charts
-    version: 1.1.3
+    version: 1.1.4
```

This needs to be the helm chart version:
```suggestion
    version: 1.2.7
```

---

So was not bumped correctly before. Went unnoticed because the kube-monitoring imports this chart directly.